.github/workflows/linux.yml

```diff
@@ -100,7 +100,8 @@ jobs:
           # Compile PDC
           mkdir $PDC_DIR
           cd $PDC_DIR
-          git checkout develop          
+          git checkout develop
```

Comment:
If we remove the checkout the action should always checkout the current branch

Reply:
ok, I'll remove it then